src/flow.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2018-2020, Michael P. Howard
+# Copyright (c) 2021-2024, Auburn University
+# Part of azplugins, released under the BSD 3-Clause License.
+
+import hoomd
+from hoomd import _hoomd
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.operation import _HOOMDBaseObject
+from . import _azplugins
+
+
+class FlowField(_HOOMDBaseObject):
+    pass
+
+
+class ConstantFlow(FlowField):
+    def __init__(self, mean_velocity):
```

Comment:
Update these all here too:
```suggestion
    def __init__(self, velocity):
```